api/public_api_v1.yaml

```diff
@@ -104,17 +91,18 @@ paths:
           description: The message cannot be delivered.
           schema:
             $ref: '#/definitions/ProblemJson'
-  '/preferences/{fiscal_code}':
-    x-swagger-router-controller: Preferences
+  '/profiles/{fiscal_code}':
     get:
-      operationId: getUserPreferences
+      operationId: getProfile
       summary: Get User's Preferences
       description: Returns the preferences for the user identified by the provided fiscal code.
```

Comment:
align description with the new 'profile' naming. 
moreover the description should explain when it returns Limited vs Extended profile.

Reply:
descriptions and operations will be my next effort (contribution welcome), I'm currently focusing on generating models